Read confirmation

aabb2002 opened the pull request:
Message shows after saving read status.

Commit:
program description completed

Commit:
added blank lines before and after book list

Commit:
read confirmation message added after saving book change

aabb2002 commented:
accidently added the read and ui files. The only change was in the main file.

claraj commented:
This has a lot of code from other issues. Make sure you check out main before creating a new branch. Check your other PRs in case they have the same issue? Suggest checking out main, make a new branch, copy only the work related to the read confirmation to the new branch, and make a new PR. Same for the other PRs that are affected

aabb2002 committed:
Merge branch 'main' into read_confirmation

Baldeaglefacts commented on main.py:
```diff
@@ -65,6 +65,7 @@ def change_read():
         new_read = ui.get_read_value()
         book.read = new_read
         book.save()
+        ui.message('You have read ' + book.title + ' by ' + book.author)
```
This doesn't check if the person is changing the status from read to unread, or vice versa. I tested the code and changed a books status from "read" to "unread", and it responded by saying "You have read Test_Title by Test_Author". It should say the opposite.

aabb2002 replied:
Got it. Now the read status is checked first before giving a message.

aabb2002 committed:
Update main.py

Change_read method now checks the books read status first before giving confirmation message.

Baldeaglefacts commented:
Works perfectly

claraj commented:
This shouldn't have been merged with the unrelated changes, see earlier note.
